cdap-ui/app/directives/complex-schema/embedded-schema-selector/embedded-schema-selector.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="clearfix"
+     ng-switch="Embedded.displayType">
+  <div class="col-xs-12"
```

Comment:
We can get to styling when we are styling the directive. Lets focus on styling in subsequent PRs. I am afraid this will cause issues when we don't have columns nested inside rows.. but that is a problem for another PR. 👍